### PR TITLE
Create text_fr.ts

### PR DESCRIPTION
**Pull request by aelisya**
Add translation for french language

### CODE REVIEW

File: src/common/i18n/text_fr.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    ExportDialog_size_L: 'grand ({0})',
+    ExportDialog_size_original: 'taille originale',
+    ExportDialog_size_custom: 'personnalisée',
+    ExportDialog_customSizeSettings: 'Maximum {0} de {1} pixels',
+    ExportDialog_customSizeSide_width: 'largeur',
+    ExportDialog_customSizeSide_height: 'hauteur',
+    ExportDialog_customSizeSide_size: 'taille',
+    ExportDialog_privacy: 'Vie privée',
+    ExportDialog_removeInfo: 'Supprimer les métadonnées',
+    ExportDialog_removeInfo_desc: 'Lorsque cette option est sélectionnée, les métadonnées telles que les EXIF-Tags seront supprimées de la photo exportée. Il s'agit d'informations sur l'appareil photo utilisé, l'endroit où la photo a été prise ou d'autres informations similaires.',
```
Comment (aelisya) [inline src/common/i18n/text_fr.ts:47]: the spelling is correct

File: src/common/i18n/text_fr.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    PhotoActionButtons_movedToTrash_one: 'Déplacement de la photo à la corbeille',
+    PhotoActionButtons_movedToTrash_more: 'Déplacement des {0} photos à la corbeille',
+    PhotoActionButtons_restoredFromTrash_one: 'Photo restaurée à partir de la corbeille',
+    PhotoActionButtons_restoredFromTrash_more: '{0} photos restaurée à partir de la corbeille',
+    PhotoActionButtons_removeFavorite: 'Supprimer des favoris',
+    PhotoActionButtons_addFavorite: 'Ajouter aux favoris',
+    PhotoActionButtons_trash: 'Déplacer à la corbeille',
+    PhotoActionButtons_restoreFromTrash: 'Restaurer de la corbeille',
+    PhotoActionButtons_restore: 'Restaurez',
+    PhotoActionButtons_photoInfo: 'Afficher les informations sur la photo sélectionnée',
```
Comment (aelisya) [inline src/common/i18n/text_fr.ts:105]: the spelling is also correct

**Comment by aelisya** [comment]
i have reviewed the spelling it's correct

**Comment by til-schneider** [comment]
Very nice, thank you very much!

The spell checker is only able to check the English language. So the translation files have to be excluded in `/.github/workflows/codespell.yml`. No problem, I can do this.

**Pull request merged**